translations/translations.json

```diff
@@ -10158,7 +10159,13 @@
       "makeAppointment": "Termin vereinbaren",
       "appointmentNecessary": "Termin erforderlich",
       "makeAppointmentTooltipWithLink": "Vereinbare an diesem Tag einen Termin über <1>die Webseite</1> oder telefonisch.",
-      "theWebsite": "die Webseite"
+      "theWebsite": "die Webseite",
+      "poisCount_zero": "0 Orte",
+      "poisCount_one": "1 Ort",
+      "poisCount_two": "{{count}} Orte",
```

Comment:
why do we need all the keys? Wouldn't one for singular and one for plural be enough and just pass the `count`